File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.py
def solution(arr1, arr2):
    answer = []
    # 정답 배열 만들기
    for i in range(len(arr1)):
        answer.append([0]*len(arr2[0]))
    
    # 행 곱셈
    for i in range(len(arr1)):
        for j in range(len(arr2[0])):
            for k in range(len(arr1[0])):
                answer[i][j] += arr1[i][k] * arr2[k][j]
    
    return answer

File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.js
function solution(arr1, arr2) {
    var answer = Array.from({length: arr1.length}, () => Array(arr2[0].length).fill(0))
    
    for (let i = 0; i < arr1.length; i++) {
        for (let j =0; j < arr2[0].length; j++) {
            for (let k = 0; k < arr1[0].length; k++)
                answer[i][j] += arr1[i][k] * arr2[k][j]
        }
    }
    return answer;
}